### PR TITLE
Create bashscript.sh to make a zip file of theme for #79

### PR DESCRIPTION
**Pull request by MatejMecka**
Don't mind the errors. Also if anyone has some improvments i'm open. 

Fixes: #79

### CODE REVIEW

File: bashscript.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# To Do. give it a better name
```
Comment (pjhampton) [inline bashscript.sh]: What about *maketheme.sh*?

Comment (pjhampton) [inline bashscript.sh]: What about `deployment_preprocess.sh`?

File: bashscript.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+cp -a ../Gramophone/. /Gramaphone/gramaphonebackup/
+
+rm -rf .*
+
```
Comment (pjhampton) [inline bashscript.sh]: Can we also delete the following: 

- docker-compose.yml
- README.md

**Comment by MatejMecka** [comment]
I fixed what you told me to. Review?

**Comment by pjhampton** [comment]
Thanks for your help @MatejMecka. Happy hacktoberfest!

**Pull request merged**